docs/portal_config.md

```diff
@@ -464,6 +464,9 @@ Below is an example, with inline comments describing what each JSON block config
           "enable": true,
           "enableDownloadManifest": true, // enables a button which allows user to download a manifest file for gen3 client
           "downloadManifestButtonText": "Download Manifest", // text to be displayed on the download manifest button
+          "enableFillRequestForm" : true, // enables a button which opens a new form to request access to a resoruce
+          "openFillRequestForm": "Request Access", // text to be displayed on fill the request form
```

Comment:
```suggestion
          "fillRequestFormDisplayText": "Request Access", // text to be displayed on fill the request form
```

Reply:
Change done, I will commit together once all of this is resolved